src/Query/Fuzzy.php

```diff
@@ -15,6 +15,16 @@
  */
 class Fuzzy extends AbstractQuery
 {
+    /**
+     * Rewrite methods: @see https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-multi-term-rewrite.html.
+     */
+    public const REWRITE_CONSTANT_SCORE = 'constant_score';
```

Comment:
Are these const always the same. I wonder if we should define these once in AbstractQuery. But then, they don't apply to all queries. What is the pattern of all the queries this applies to? Should we have an abstract query or similar that all these extend?

Reply:
These constants are always the same. The only pattern between these, is that they are all Term-level queries that use wildcards in some shape or form.

When writing the PR I was also debating puttings the constants in AbstractQuery but reached the same conclusion.

I guess we could have a new level of abstract query for these. Not sure how to name it though.